src/Features/CSharp/Portable/CodeFixes/AddExplicitCast/CSharpAddExplicitCastCodeFixProvider.cs

```diff
@@ -0,0 +1,219 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
```

Comment:
can you change this name back to AddExplicitCastCodeFixProvider.cs.  that way we can see the diff in the PR.  after the PR goes through you can do the file name change.

Reply:
yeah sure